src/pages/index.js

```diff
@@ -51,7 +51,7 @@ class Home extends Component {
     return (
       <Layout location={location}>
         <TitleAndMetaTags
-          title="React &ndash; A JavaScript library for building user interfaces"
+          title="React &ndash; Uma biblioteca JavaScript para criar user interfaces"
```

Comment:
```suggestion
          title="React &ndash; Uma biblioteca JavaScript para criar interfaces do usuário"
```

"user interfaces" -> interfaces do usuário? 

Não me parece ruim, acha que precisa deixar em inglês?

Reply:
Fiquei em dúvida também e resolvi deixar em inglês. Tava pensando em traduzir como "interfaces **de** usuário" [[1]](https://pt.wikipedia.org/wiki/Interface_do_utilizador)

Reply:
concordo que "interfaces **de** usuário" ficaria melhor